docs/devices/2430-100.md

```diff
@@ -17,10 +17,11 @@ description: "Integrate your Gira 2430-100 via Zigbee2mqtt with whatever smart h
 
 ## Notes
 
-None
+How do you pair it?
```

Comment:
Do you know?

Reply:
@pinguinpfleger - the pairing procedure is described in the Gira user guide: https://partner.gira.de/data3/24301390.pdf (chapter 4.3 "Adding the transmitter to an existing network").